Cargo.lock

```diff
@@ -39,9 +39,9 @@ dependencies = [
 
 [[package]]
 name = "aho-corasick"
-version = "1.0.2"
+version = "1.0.4"
```

Comment:
I think the commit message should make clear that it is more than just `serde` and `serde_derive` that is being updated.